tests/Testcontainers.Tests/Fixtures/Containers/Unix/Modules/Databases/Neo4jFixture.cs

```diff
@@ -0,0 +1,43 @@
+﻿namespace DotNet.Testcontainers.Tests.Fixtures
+{
+  using System.Threading.Tasks;
+  using DotNet.Testcontainers.Builders;
+  using DotNet.Testcontainers.Configurations;
+  using DotNet.Testcontainers.Containers;
+  using JetBrains.Annotations;
+  using Neo4j.Driver;
+
+  [UsedImplicitly]
+  public sealed class Neo4jFixture : DatabaseFixture<Neo4jTestcontainer, IDriver>
+  {
+    private readonly TestcontainerDatabaseConfiguration configuration = new Neo4jTestcontainerConfiguration { Username = "username", Password = "password" };
```

Comment:
It does not make sense to set the `Username` property based on your implementation.